### ISSUE TITLE
*rate.set_kb get applied in KiloBytes but it is shown in wiki as kb?

### ISSUE THREAD

**Issue by rezad1393** [rakshasa/rtorrent#1150]
in the wiki the notation used for this is kbs which usually means kilobits per seconds.
but I used this in schedule command and it gets set as kBs as in kilo BYTES per seconds.

**Comment by kannibalox** [comment]
I agree it's a little confusing, but the commands are standardized on being lowercase, and it might be even more confusing if this was the lone exception. It's also actually set in kibibytes per second (KiB/s).

**Comment by rezad1393** [comment]
> I agree it's a little confusing, but the commands are standardized on being lowercase, and it might be even more confusing if this was the lone exception. It's also actually set in kibibytes per second (KiB/s).

sorry. sometimes I write something and think that the reader knows what I think and then I skip some stuff.

what I mean was that maybe the WIKI documentation should be fixed. not the program.
